server/http/http.go

```diff
@@ -15,6 +15,7 @@
 package http
 
 import (
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

Comment:
run `goimports` it will sort the imports, if you see it is not working then remove spaces and run it again.

Reply:
Done.